tests/Api.Tests/Endpoints/AuthenticatorsEndpointsTests.cs

```diff
@@ -0,0 +1,201 @@
+using System.Collections.Immutable;
+using AutoFixture;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Passwordless.Api.Endpoints;
+using Passwordless.Common.Models.Authenticators;
+using Passwordless.Service;
+using Passwordless.Service.Features;
+using Passwordless.Service.Helpers;
+
+namespace Passwordless.Api.Tests.Endpoints;
+
+public class AuthenticatorsEndpointsTests
+{
+    private readonly Fixture _fixture = new();
+
+    #region ListConfiguredAuthenticatorsAsync
+    [Fact]
+    public async Task ListConfiguredAuthenticatorsAsync_Throws_Forbidden_WhenAttestationIsNotAllowed()
+    {
+        // Arrange
+        var request = _fixture.Create<ConfiguredAuthenticatorRequest>();
```

Comment:
Are these tests needed now that `AuthenticatorsTests` are in place? Do they cover something else?

Reply:
Yeah, I use them to quickly verify some expected behavior